web_services/foxx/brightid/initdb.js

```diff
@@ -206,7 +206,20 @@ function v5_5() {
   }
 }
 
-const upgrades = ['v5', 'v5_3', 'v5_5'];
+function v5_6() {
+  console.log("removing 'verification' attribute form contexts collection");
+  const contextsColl = arango._collection('contexts');
+  query`
+    FOR doc IN ${contextsColl}
+      REPLACE doc WITH UNSET(doc, 'verification') IN ${contextsColl}`;
+
+  console.log("removing 'ethName' attribute form context collection");
```

Comment:
`const ETHNAME_NOT_SET = 8;` should also be removed from `index.js` file

Reply:
It's done.